package.json

```diff
@@ -200,67 +210,69 @@
     "babel-polyfill": "6.13.0",
     "chalk": "1.1.3",
     "compression": "1.6.2",
+    "cross-env": "2.0.0",
     "express": "4.14.0",
-    "fontfaceobserver": "2.0.1",
-    "history": "3.0.0",
+    "fontfaceobserver": "2.0.4",
     "immutable": "3.8.1",
     "intl": "1.2.4",
     "invariant": "2.2.1",
     "ip": "1.1.3",
     "lodash": "4.15.0",
-    "react": "15.3.0",
-    "react-dom": "15.3.0",
+    "minimist": "1.2.0",
+    "react": "15.3.1",
+    "react-dom": "15.3.1",
     "react-helmet": "3.1.0",
+    "react-intl": "2.1.5",
     "react-redux": "4.4.5",
-    "react-intl": "2.1.3",
-    "react-router": "2.6.1",
+    "react-router": "2.7.0",
     "react-router-redux": "4.0.5",
-    "react-router-scroll": "0.2.1",
-    "redux": "3.5.2",
-    "redux-immutable": "3.0.7",
-    "redux-saga": "0.11.0",
+    "react-router-scroll": "0.3.2",
+    "redux": "3.6.0",
+    "redux-immutable": "3.0.8",
+    "redux-saga": "0.11.1",
     "reselect": "2.5.3",
     "sanitize.css": "4.1.0",
     "warning": "3.0.0",
     "whatwg-fetch": "1.0.0"
   },
   "devDependencies": {
-    "babel-cli": "6.11.4",
-    "babel-core": "6.13.2",
+    "babel-cli": "6.14.0",
+    "babel-core": "6.14.0",
     "babel-eslint": "6.1.2",
-    "babel-loader": "6.2.4",
-    "babel-plugin-react-intl": "2.1.3",
+    "babel-loader": "6.2.5",
+    "babel-jest": "^15.0.0",
+    "babel-plugin-react-intl": "2.2.0",
     "babel-plugin-react-transform": "2.0.2",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.11.5",
     "babel-plugin-transform-react-constant-elements": "6.9.1",
     "babel-plugin-transform-react-inline-elements": "6.8.0",
     "babel-plugin-transform-react-remove-prop-types": "0.2.9",
-    "babel-preset-es2015": "6.13.2",
+    "babel-preset-es2015": "6.14.0",
     "babel-preset-react": "6.11.1",
     "babel-preset-react-hmre": "1.1.1",
     "babel-preset-stage-0": "6.5.0",
     "chai": "3.5.0",
-    "chai-enzyme": "0.5.0",
-    "cheerio": "0.20.0",
+    "chai-enzyme": "0.5.1",
+    "cheerio": "0.22.0",
     "coveralls": "2.11.12",
-    "cross-env": "2.0.0",
-    "css-loader": "0.23.1",
+    "css-loader": "0.24.0",
     "enzyme": "2.4.1",
-    "eslint": "3.3.0",
+    "eslint": "3.4.0",
     "eslint-config-airbnb": "10.0.1",
     "eslint-import-resolver-webpack": "0.5.1",
-    "eslint-plugin-import": "1.12.0",
-    "eslint-plugin-jsx-a11y": "2.1.0",
-    "eslint-plugin-react": "6.0.0",
+    "eslint-plugin-import": "1.14.0",
+    "eslint-plugin-jsx-a11y": "2.2.1",
+    "eslint-plugin-react": "6.2.0",
     "eventsource-polyfill": "0.9.6",
     "exports-loader": "0.6.3",
-    "extract-text-webpack-plugin": "1.0.1",
+    "extract-text-webpack-plugin": "2.0.0-beta.4",
     "file-loader": "0.9.0",
     "html-loader": "0.4.3",
     "html-webpack-plugin": "2.22.0",
     "image-webpack-loader": "2.0.0",
     "imports-loader": "0.6.5",
     "jest": "^15.1.1",
```

Comment:
remove `jest` as you have `jest-cli` installed to now

Looking at the repos for both. `jest` is just a wrapper for `jest-cli`. So go with `jest-cli`

[jest-cli](https://github.com/facebook/jest/blob/ec782b567bf343d9d278cd79265a1af2bfc3e83c/packages/jest-cli/package.json)

[jest](https://github.com/facebook/jest/blob/ec782b567bf343d9d278cd79265a1af2bfc3e83c/packages/jest/package.json)